server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -530,6 +531,9 @@ private void run() {
 
     ProbabilitySampler sampler = new ProbabilitySampler(getConfiguration().getFraction(Property.GC_TRACE_PERCENT));
 
+    // Map of tserver -> timestamp, used by the GCWriteAheadLogs to track state of when a tablet server went down
+    Map<HostAndPort, Long> firstSeenDead = new HashMap<>();
```

Comment:
Nit: let's make this `final` to make sure that we don't ever accidentally overwrite the variable and end up with the same bug in the future.